setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
I think some docstrings from Cython end up in the docs.  Here is one example I found for the `swt_max_level` routine which is defined in `pywt/_extensions/_swt.pyx` and then imported into `__all__` in  `pywt/_swt.py`.  It appears to be getting rendered correctly here:

http://pywavelets.readthedocs.io/en/latest/ref/swt-stationary-wavelet-transform.html#maximum-decomposition-level-swt-max-level